pkg/policy/api/egress.go

```diff
@@ -148,12 +148,37 @@ type EgressRule struct {
 	//     - 'sg-XXXXXXXXXXXXX'
 	// +optional
 	ToGroups []ToGroups `json:"toGroups,omitempty"`
+
+	aggregatedSelectors EndpointSelectorSlice
```

Comment:
We are mixing the API with implementation details. This structure should kept somewhere else.

Reply:
I fully agree. However, this was here before, and refactoring to an internal `IngressRule` and `EgressRule` is a bit too much for this cycle. Opened a new Issue #8353 to track this. Also added TODO comments here with reference to the GH issue.